test/mason/publish-tests/noFork.good

```diff
@@ -1 +1,9 @@
+Checking Registry with DummyPath path.
+------------------------------------------------------
+The current mason environment is:
+MASON_HOME: /home/anking/chapel/test/mason/publish-tests/mason_home *
```

Comment:
The issue is with this path. See the `test/mason/PREDIFF` for replacing the path with a string `$PWD`.